runtime/vara/src/weights/pallet_airdrop.rs

```diff
@@ -73,18 +75,20 @@ impl WeightInfo for () {
         // Proof Size summary in bytes:
         //  Measured:  `103`
         //  Estimated: `6196`
-        // Minimum execution time: 24_965_000 picoseconds.
-        Weight::from_parts(26_088_278, 6196)
+        // Minimum execution time: 25_544_000 picoseconds.
+        Weight::from_parts(26_559_640, 6196)
             .saturating_add(RocksDbWeight::get().reads(2_u64))
             .saturating_add(RocksDbWeight::get().writes(2_u64))
     }
```

Comment:
**Suggestion:** The execution time for the functions `claim` and `transfer_vested` have been increased. It's recommended to investigate the reason for this increase and try to optimize the functions if possible.
```suggestion
        The code optimization should be done based on the specific logic of these functions. It's not possible to provide a generic optimized code snippet.
```